spec/support/fixtures/foo

```diff
@@ -25,7 +25,7 @@ module Foo
 
       class Console < Hanami::Cli::Command
         desc "Starts Foo console"
-        option :engine, desc: "Force a console engine: (irb/pry/ripl)"
+        option :engine, desc: "Force a console engine", list: %w(irb pry ripl)
```

Comment:
@AlfonsoUceda `list:` as the option name here feels a little funny to me... what do you think of something like `values:`?